modules/internal/DefaultRectangular.chpl

```diff
@@ -868,9 +868,16 @@ module DefaultRectangular {
     }
   
     // only need second version because wrapper record can pass a 1-tuple
-    inline proc dsiAccess(ind: idxType ...1) ref where rank == 1
+    inline proc dsiAccess(ind: idxType ...1) ref
+    where rank == 1
```

Comment:
I put them on separate lines because
 1) otherwise the line would be too long and
 2) the 'where' clause is at least as important as 'inline' and my eye focuses more on the left.

I'm open to other formatting suggestions. Just putting them on one line would be > 80 characters, so I don't think that's a good option.